pkgs/development/python-modules/werkzeug/default.nix

```diff
@@ -15,17 +16,21 @@
 
 buildPythonPackage rec {
   pname = "werkzeug";
-  version = "2.2.3";
-  format = "setuptools";
+  version = "2.3.6";
+  format = "pyproject";
 
   disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     pname = "Werkzeug";
     inherit version;
-    hash = "sha256-LhzMlBfU2jWLnebxdOOsCUOR6h1PvvLWZ4ZdgZ39Cv4=";
+    hash = "sha256-mMd03y+RsFVQB4iR3uXw6wy3l6Uix1eiRSuc7lsgIzA=";
   };
 
+  nativeBuildInputs = [
+    setuptools
+  ];
```

Comment:
Can you also add wheel to this? It's implicitly added right now by our Python build hooks, but I'm working on changing this.

Reply:
fixed in [force-push](https://github.com/NixOS/nixpkgs/compare/77adbeb69e1da83bc8f87e05667345fbbc00aa77..c17fc82201157b2f00e5520052ac8de30062f78c)